pyravendb/commands/raven_commands.py

```diff
@@ -109,7 +109,7 @@ def create_request(self, server_node):
             # If it is too big, we drop to POST (note that means that we can't use the HTTP cache any longer)
             if (sum(len(x) for x in key_or_keys)) > 1024:
                 self.method = "POST"
-                self.data = list(key_or_keys)
+                self.data = dict(Ids=list(key_or_keys))
```

Comment:
Can we adjust spacing around `=`?

Reply:
There is the general recommended `Python` formatting - `PEP8`.
We should probably use something like https://pypi.org/project/autopep8/ to ensure consistent formatting.

Reply:
Yup, something to run with github actions on PR.